### PR TITLE
test/runtime: migrate 13-fd-open.sh to Ginkgo

### PR DESCRIPTION
**Pull request by ianvernon**
Signed-off by: Ian Vernon <ian@cilium.io>

Relates-to: #1839

### CODE REVIEW

**Comment by ianvernon** [comment]
test-me-please

File: test/runtime/chaos.go [hunk 1]
```diff
@@ -109,4 +110,14 @@ var _ = Describe("RuntimeChaos", func() {
...
 			"Some network interfaces were accidentally removed!")
 	}, 300)
+
+	It("Checking for file-descriptor leak", func() {
+		threshold := 5000
+		fds, err := vm.Exec("sudo lsof -p `pidof cilium-node-monitor` -p `pidof cilium-agent` -p `pidof cilium-docker` 2>/dev/null | wc -l").IntOutput()
+		Expect(err).Should(BeNil())
+
+		aboveThreshold := fds > threshold
+		Expect(aboveThreshold).Should(Equal(false),
```
Comment (eloycoto) [inline test/runtime/chaos.go]: I don't really like this approach, what about `Expect(fds).To(BeNumerically("<", threshold))` and you'll get the values in the assert message. 

Regards

**Review by eloycoto — CHANGES_REQUESTED** [review]

File: test/runtime/chaos.go (hunk 1 above)
Comment (ianvernon) [reply test/runtime/chaos.go]: I was looking for something like that, but couldn't find it. Thanks for that, I prefer what you suggested as well :-)

Comment (ianvernon) [reply test/runtime/chaos.go]: Fixed in latest upload.

**Comment by ianvernon** [comment]
test-me-please

File: test/runtime/chaos.go [hunk 2]
```diff
@@ -109,4 +110,13 @@ var _ = Describe("RuntimeChaos", func() {
...
 			"Some network interfaces were accidentally removed!")
 	}, 300)
+
+	It("Checking for file-descriptor leak", func() {
+		threshold := 5000
+		fds, err := vm.Exec("sudo lsof -p `pidof cilium-node-monitor` -p `pidof cilium-agent` -p `pidof cilium-docker` 2>/dev/null | wc -l").IntOutput()
+		Expect(err).Should(BeNil())
+
+		Expect(fds).To(BeNumerically("<", threshold),
+			fmt.Sprintf("%d file descriptors open from Cilium processes", fds))
```
Comment (eloycoto) [inline test/runtime/chaos.go:120]: Just FYI: You can do like this: 
```
Expect(fds).To(BeNumerically("<", threshold), "%d file descriptors open from Cilium processes", fds))
```

Sprintf is not needed at all ;-)

**Review by eloycoto — APPROVED** [review]

**Pull request merged**